### PR TITLE
Upgrade to Decidim v0.25

### PR DESCRIPTION
**Pull request by LauraJaimeC**
CHANGELOG: https://github.com/decidim/decidim/blob/release/0.25-stable/CHANGELOG.md

- [x] Merge before:
https://github.com/CodiTramuntana/decidim-file_authorization_handler/pull/2
https://github.com/CodiTramuntana/decidim-members/pull/13
https://github.com/CodiTramuntana/decidim-verifications-csv_emails/pull/15
https://github.com/CodiTramuntana/decidim-verifications-sant_boi_census/pull/4

After deployment we must remember to execute some tasks:
- the migration from `carrierwave` to `activestorage` by executing the rake task (see https://github.com/decidim/decidim/blob/release/0.25-stable/CHANGELOG.md#activestorage-migration):
```
rails decidim:active_storage_migrations:migrate_from_carrierwave_to_active_storage
```
- Adding User groups in global search `Decidim::UserGroup.find_each(&:try_update_index_for_search_resource)`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,17 +4,17 @@ source "https://rubygems.org"
...
 gem "decidim-initiatives", DECIDIM_VERSION
-gem "decidim-members", git: "https://github.com/CodiTramuntana/decidim-members.git", tag: "v0.1.18"
+gem "decidim-members", git: "https://github.com/CodiTramuntana/decidim-members.git", branch: "0.25-stable"
 gem "decidim-sortitions", DECIDIM_VERSION
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "0.24-stable"
-gem "decidim-verifications-csv_email", git: "https://github.com/CodiTramuntana/decidim-verifications-csv_emails.git", tag: "v0.0.9"
-gem "decidim-verifications-sant_boi_census", git: "https://github.com/CodiTramuntana/decidim-verifications-sant_boi_census.git", tag: "v0.0.2"
+gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer"
+gem "decidim-verifications-csv_email", git: "https://github.com/CodiTramuntana/decidim-verifications-csv_emails.git", branch: "0.25-stable"
+gem "decidim-verifications-sant_boi_census", git: "https://github.com/CodiTramuntana/decidim-verifications-sant_boi_census.git", branch: "0.25-stable"
```
Comment (tramuntanal) [inline Gemfile:17]: we must review this and use tags for latest instead of branches

File: Gemfile [hunk 2]
```diff
@@ -25,6 +25,8 @@ gem "whenever", require: false
 
 gem "figaro", ">= 1.1.1"
 gem "openssl"
+# For seeds
+gem "faker"
```
Comment (tramuntanal) [inline Gemfile:29]: we don't want faker in production

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Good job @LauraJaimeC we must just check versioning of our imported modules

File: Gemfile (hunk 1 above)
Comment (LauraJaimeC) [reply Gemfile:17]: There are the PRs in the description pending review, can you take a look? thank you!

**Review by tramuntanal — APPROVED** [review]
LGTM
Good job @LauraJaimeC !

**Pull request merged**